test.sh

```diff
@@ -65,6 +65,16 @@ rm java/examples/binary/deployments/*
 
 test_container "s2i-java-binary-example"
 
+
+# ----------------------------------------------------------------------------------
+# Spring Boot Developer Tools
+# ----------------------------------------------------------------------------------
+
+s2i build --copy java/examples/spring-devtools fabric8/s2i-java fabric8/s2i-java-spring-devtools-example
+
+test_container "s2i-java-spring-devtools-example"
```

Comment:
You must now check that the build log contains a certain string that the fat JAR has been exploded